app/helpers/application_helper.rb

```diff
@@ -13,4 +13,9 @@ def markdown_to_html(markdown)
     redcarpet = Redcarpet::Markdown.new(renderer, extensions)
     (redcarpet.render markdown).html_safe
   end
+
+  def sortable(column, title = nil)
+    title ||= column
+    link_to title, {:sort => column}
```

Comment:
Review the other options available on the `link_to` method here as they are worth being reminded of: http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to